modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring graceful node shutdown
+
+To configure graceful node shutdown, create a `KubeletConfig` custom resource (CR) to specify a shutdown grace period for pods on the node. The graceful node shutdown feature minimizes interruption to workloads running on these pods.
+
+[NOTE]
+====
+If you do not configure graceful node shutdown, the default grace period is `0` and the pod is forcefully evicted from the node.
+====
+
+.Prerequisites
+
+* You have access to the cluster with the `cluster-admin` role.
+* You have defined priority classes for pods that require critical or regular classification.
+
+.Procedure
+
+. Define shutdown grace periods in the `KubeletConfig` CR by saving the following YAML in the `kubelet-gns.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: graceful-shutdown
+  namespace: openshift-machine-config-operator
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+  kubeletConfig:
+    shutdownGracePeriod: "3m" <2>
+    shutdownGracePeriodCriticalPods: "2m" <3>
+----
+<1> This example applies shutdown grace periods to nodes with the `worker` role. 
+<2> Define a time period for regular pods to shut down.
+<3> Define a time period for critical pods to shut down. 
+
+. Create the KubeletConfig CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f kubelet-gns.yaml
+----
++
+.Example output
+[source,terminal]
+----
+kubeletconfig.machineconfiguration.openshift.io/graceful-shutdown created
+----
+
+.Verification
+
+. View the kubelet logs for a specific node by running the following command, replacing `<node_name>` with the name of the node:
```

Comment:
I think a better way to verify the feature take effect is : oc debug node/XX and check /etc/kubernetes/kubelet.conf, the shutdownGracePeriod and shutdownGracePeriodCriticalPods are as expected.
Your method is also acceptable.